styles/styles.css

```diff
@@ -369,6 +806,56 @@
 
     /* Block specific */
     --signup-padding: var(--spacing-52);
+
+    /* numbers */
+    --border-radius-none: 0rem;
+    --border-radius-small: 0.125rem;
+    --border-radius-base: 0.25rem;
+    --border-radius-medium: 0.5rem;
+    --border-radius-large: 1rem;
+    --border-radius-x-large: 2rem;
+    --global-gutter: 1.5rem;
+    --global-section-padding: 5rem;
+    --grid-margin: 4.5rem;
+    --spacing-none: 0rem;
+    --spacing-xtiny: 0.125rem;
+    --spacing-tiny: 0.25rem;
+    --spacing-xxsmall: 0.5rem;
+    --spacing-xsmall: 0.75rem;
+    --spacing-small: 1rem;
+    --spacing-regular: 1.5rem;
+    --spacing-medium: 2rem;
+    --spacing-large: 2.5rem;
+    --spacing-xlarge: 3rem;
+    --spacing-xxlarge: 4rem;
+    --spacing-huge: 5rem;
+    --spacing-xhuge: 6rem;
+    --size-icon-small: 0.875rem;
+    --size-icon-regular: 1rem;
+    --size-icon-large: 1.25rem;
+    --size-icon-xxlarge: 6rem;
+
+    /* Typography */
+    --font-size-100: 0.75rem;
+    --font-size-200: 0.875rem;
+    --font-size-300: 1rem;
+    --font-size-400: 1.125rem;
+    --font-size-500: 1.25rem;
+    --font-size-600: 1.5rem;
+    --font-size-700: 1.75rem;
+    --font-size-800: 2rem;
+    --font-size-900: 2.25rem;
+    --font-size-1000: 2.5rem;
+    --font-size-1100: 3rem;
+    --font-size-1200: 3.75rem;
+
+    /* string */
+    --body-font-weight-bold: bold;
+    --body-font-weight-italic: italic;
+    --body-font-weight-regular: regular;
+    --global-font-familly: 'Open Sans';
```

Comment:
Shouldn't we specify a default global font family here in case the custom **OpenSans** font fails to load?